### ISSUE TITLE
[1.7.10] Mirroring Signs

### ISSUE THREAD

**Issue by Pilzinsel64** [Gollorum/Signpost#107]
Hello!
I would like to request a single additional feature for the 1.7.10 version: Mirroring sings.
At the moment the sign always directs to left. But it would be really helpful to be able to change the direction to right.

Not sure if you're still maintaining the 1.7.10 version, but hey, I can try suggesting it. :)
I also would implement it myself, but not sure where to touch. You know the source, I would need to get into first and I'm still now to mod development. And I'm not sure if it can be done e.g. via a single parameter or if it's needed to clone all models and mirror it, add a new property and handle a new interaction (e.g. shift right click or whatever).

PS: This is a really great mod, also for 1.7.10! :) Was hard to find such a mod. Except the mirrored signs it has everything I need.

**Comment by Gollorum** [comment]
Hey,
I'm in fact not really maintaining that version, but you're in luck: The feature should already exist 😄 
If I recall correctly, you have to Shift-Left-Click (hit while sneaking) the sign with the Wrench.

**Comment by Pilzinsel64** [comment]
Uh? Will try again later! Thanks for the hint. 😄 
I thought I already tried every common key combination and only got it to rotate. Maybe I missed one.
And if it doesn't work, I'm going to take a look into the sorrce again. :)

Btw, yesterday I forked the 1.7.10 Branche an migrated to an up-to-date build script, applieded unique Code Format, updated dependencies and added GitHub workflows.
I did that to keep the source build able for me for my private modpack. I hope this is ok. If not just tell me, I then make the repo private.
If you ever want, of course I can open a PR with that changes whenever you want.
